NAMESPACE

```diff
@@ -3,10 +3,8 @@
 S3method(print_html,default)
 export(off)
 export(on)
+export(path)
 export(print_html)
-export(pull_obj)
-export(push_obj)
-export(tv_remote)
-export(tv_remote_own_session)
+export(status)
```

Comment:
`get_status()` or `tv_get_status()`? Should all functions use a prefix, or do we expect the users to never attach the package?

Reply:
No prefix for anything. The only exported functions are now `on()`, `off()`, and `status()`, all of them usually used without attaching.

I feel that short `status()` function sound nicer together with `on()` and `off()`. Nicer than `get_status()`, and tells the same.